gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -42,7 +42,7 @@ label.coursegrade.studentnotreleased=Not Yet Released
 
 label.gradeitem.externalapplabel=Gradebook Item coming from {0}
 label.gradeitem.released=Gradebook Item has been released and is visible to students.
-label.gradeitem.notreleased=Gradebook item is not visible to students.
+label.gradeitem.notreleased=Released Gradebook items are not visible to students.
```

Comment:
I think the intention of SAK-40641 is to leave the original message alone (it is the tooltip for the hidden icon in a gradebook column), and to add a second message at the top of the screen to indicate that even the "released" (ie. not hidden) items will still not be visible to students because of the "Display released Gradebook items to students" is not checked in the Gradebook Settings. So there is more to this ticket than just wording change.

Reply:
I've been watching but I dont know where do you want the new message to appear, and what kind message is, popup, alert, or at the same table?

Reply:
You should ask for clarification on the Jira ticket to be sure, but it seems to me that the original request is for a banner message at the top of the Gradebook page that indicates the status of the "Display released Gradebook items to students" setting.

Reply:
Okay, thanks for your help